ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -401,22 +365,10 @@ class ProfileMainScreen extends React.PureComponent<Props> {
             <Markdown>{I18n.t("profile.main.contextualHelp")}</Markdown>
           )
         }}
-        banner={ExperimentalFeaturesBanner}
```

Comment:
if you decide for the first proposed approach, it could be maintained for rendering the banner once new experimental feature are added

Reply:
Yes I'm agree with you. Indeed I just passed **undefined** to banner. Are there better solutions?

Reply:
Maybe it could be better to just leave it as `banner={ExperimentalFeaturesBanner}`
Once experimental feature will be included again, it will be required to enable again the switch only. Does it make sense?